src/sentry/tasks/post_process.py

```diff
@@ -1144,14 +1144,17 @@ def process_code_mappings(job: PostProcessJob) -> None:
             org_slug = org.slug
             next_time = timezone.now() + timedelta(hours=1)
 
+            extra: dict[str, Any] = {
```

Comment:
nit: might want to pull this inside the `if features.has("organizations:derive-code-mappings", org):` block since it's only used there